aioftp/client.py

```diff
@@ -45,7 +47,8 @@ class Code(str):
     """
     Representation of server status code.
     """
-    def matches(self, mask):
+
+    def matches(self, mask: str):
```

Comment:
We should add return types of all functions, to have strict mypy checks

Reply:
OK ,I'll double check later.